NHSN/src/test/java/com/nhsn/pages/ViewLineLevelDataPage.java

```diff
@@ -0,0 +1,63 @@
+package com.nhsn.pages;
+
+import org.openqa.selenium.*;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.asserts.SoftAssert;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.time.Instant;
+import java.util.*;
+
+public class ViewLineLevelDataPage {
+    private WebDriver driver;
+    private WebDriverWait wait;
+
+    public ViewLineLevelDataPage(WebDriver driver) {
+        this.driver = driver;
+        wait = new WebDriverWait(driver, 20);
+    }
+    public void clickOnExcludeSelectedButton()
+    {
+       WebElement excludeButton = driver.findElement(By.xpath("//button[contains(text(),'Exclude Selected')]"));
+       excludeButton.click();
+       wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath("//button[text()=' Close ']")));
+    }
+
+    public void clickOnExcludeButtonOnFirstRow()
+    {
+        WebElement firstRow = driver.findElement(By.xpath("//h6[text()='Patients']/following-sibling::table[1]//tbody/tr[1]"));
+        firstRow.findElement(By.xpath("//button[text()='Exclude']")).click();
```

Comment:
I'd like you to keep an eye out for these type of XPath statements...
It would be better if this were `By.css("button.exclude")` but that requires putting a class or id on each of the buttons.
We can use ID when the buttons don't repeat, but in this "Exclude" button scenario, we would need to use a CSS class to identify the button because it *does* repeat.
In any event, the above selector would be better than looking for a button whose text matches "Exclude"... Then we can change the text without having to alter the regression tests.
In order to make this work, though, we would need a list of these types of `.click()` selectors, so that we can go into the application code, modify the HTML, and then tell you what to look for instead.
Please consider going through your regression test code and finding cases where this would be beneficial.